packages/create-keystone-app/templates/todo/package.json.ejs

```diff
@@ -2,7 +2,7 @@
   "name": "<%= appName%>",
   "version": "0.0.0",
   "scripts": {
-    "start": "NODE_ENV=development DISABLE_LOGGING=true keystone"
+    "start": "cross-env NODE_ENV=development DISABLE_LOGGING=true keystone"
   },
   "private": true,
   "dependencies": {
```

Comment:
`cross-env` needs to be in the dependencies here, not in create-keystone-app's dependencies.